keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/VerificationRepositoryTests.kt

```diff
@@ -172,6 +174,50 @@ abstract class VerificationRepositoryTests<IMPLEMENTATION : VerificationReposito
       .with(VerificationState::metadata) { isEqualTo(metadata) }
   }
 
+  @Test
+  fun `successive updates can add and update metadata`() {
```

Comment:
This tests that we can add fields to the metadata & it will merge with anything already there